pom.xml

```diff
@@ -27,6 +27,20 @@
 			<artifactId>spring-boot-starter-test</artifactId>
 			<scope>test</scope>
 		</dependency>
+		<!-- https://mvnrepository.com/artifact/org.junit.jupiter/junit-jupiter-api -->
+		<dependency>
```

Comment:
Ensure that the 'junit-jupiter-api' dependency is necessary as 'spring-boot-starter-test' might already include it.